docs/source/getting_started.rst

```diff
@@ -1,55 +1,60 @@
-================
+===============
 Getting Started
-================
+===============
 
-Overview
----------
+Start by importing FURY.
 
-Free Unified Rendering for Python
+.. code-block:: python
 
+    import numpy as np
+    from fury import window, actor, ui, io, utils
 
-What is Fury ?
----------------
+To import a model, use ``io.load_polydata()``. Currently supported formats include OBJ, VTK, FIB, PLY, STL and XML.
+Let us include the Suzannemodel used by blender
 
+.. code-block:: python
 
+    suzanne = io.load_polydata('models/suzanne.obj')
+    suzanne = utils.get_polymapper_from_polydata(suzanne)
+    suzanne = utils.get_actor_from_polymapper(suzanne)
 
-Mission Statement
-------------------
+Set the Opacity of the model::
 
-The purpose of FURY is to make it easier to do better diffusion MR imaging research. Following up with the nipy mission statement we aim to build software that is
+    modelsuzanne.GetProperty().SetOpacity(0.5)
 
-* clearly written
-* clearly explained
-* a good fit for the underlying ideas
-* a natural home for collaboration
+Let's create random variables for cylinder parameters
 
-We hope that, if we fail to do this, you will let us know and we will try and make it better.
+.. code-block:: python
 
-Features
----------
+    centres = np.random.rand(2,3)
+    directions = np.random.rand(2,3)
+    heights = np.random.rand(2)
+    colors = np.random.rand(2,3)
 
-**Efficient**
+Now, we Create a cylinder::
 
+    cylinders = actor.cylinder(centres,directions,colors,heights=heights)
 
+We create a scene. Everything to be rendered is to be added to the scene::
 
-**Robust**
+    scene = window.Scene()
 
+We set window scene variable Eg: (width, height)::
```

Comment:
We set window scene variable Eg. (width, height):: -> We set the window scene variables e.g. (width, height)::